### PR TITLE
Fixed the ensembl transcript gene_id and msigdb geneSet publication

### PR DESCRIPTION
**Pull request by evergreen700**
Unsure if the publication ID is the format you wanted but at least it's attached to the record now. I'll probably look at another record with publications next and make sure the numbers are able to match each other.

### CODE REVIEW

**Comment by evergreen700** [comment]
Pubmed:
- Changed the transform.yaml to be more flexible with different absrtract formats
- Changed the download files to work a little bit better. There is still a snakemake bug in there but that would probably be better addressed within the lathe source code.

**Comment by evergreen700** [comment]
Cellosarus:
- Obo reader modified slightly to capture species name but shouldn't affect execution of sifter file
- Sifter file modified to generate a "species" field that it then filters the samples by. This field is then discarded

**Comment by evergreen700** [comment]
BindingDB:
- Download.yaml fixed by removing name inconsistencies
- tsvParser.py created to handle the repeating columns at the end of the file
- transformTsv.yaml modified to read protien id's as a list instead of a single string, allows multiple protiens to be connected to a binding record
- protein_compound_association modified to add protein_structure property.

File: transform/bindingdb/.ipynb_checkpoints/download-checkpoint.yaml
Comment (kellrott) [inline transform/bindingdb/.ipynb_checkpoints/download-checkpoint.yaml:1]: Why is this file in the ipython notebook checkpoint directory?

File: transform/bindingdb/.ipynb_checkpoints/download-checkpoint.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+
+
+class: lathe
+name: bindingdbDownload
+
+scripts:
```
Comment (kellrott) [inline transform/bindingdb/.ipynb_checkpoints/download-checkpoint.yaml]: Another file in the ipython checkpoint directory.

Comment (kellrott) [inline transform/bindingdb/.ipynb_checkpoints/download-checkpoint.yaml]: I'm trying to move download commands to the `prep` stanzas

File: transform/bindingdb/.ipynb_checkpoints/transform_tsv-checkpoint.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  schema: ../../schema/
+
+inputs:
+  bindingdb:
+    tableLoad:
+      input: "{{config.tsv}}"
+
+pipelines:
+  debug:
+    - from: bindingdb
```
Comment (kellrott) [inline transform/bindingdb/.ipynb_checkpoints/transform_tsv-checkpoint.yaml]: Remember to clear out debug outputs once the script is done

File: transform/bindingdb/.ipynb_checkpoints/tsvParse-checkpoint.py
Comment (kellrott) [inline transform/bindingdb/.ipynb_checkpoints/tsvParse-checkpoint.py:1]: Is this file an artifact that was accidentally checked in?

File: transform/bindingdb/transform_tsv.yaml [hunk 3]
```diff
@@ -2,22 +2,18 @@
...
   tsv: ../../source/bindingdb/BindingDB_All.tsv
-  schema: ../../schema/
+  schema: ../../schema
 
 inputs:
   bindingdb:
-    tableLoad:
-      input: "{{config.tsv}}"
+    plugin:
+      commandLine: ./tsvParse.py {{config.tsv}}
```
Comment (kellrott) [inline transform/bindingdb/transform_tsv.yaml]: Lets put scripting in the `util` directory

File: transform/bindingdb/tsvParse.py
Comment (kellrott) [inline transform/bindingdb/tsvParse.py:1]: Move this to `util/`

File: transform/pubmed/download.yaml [hunk 4]
```diff
@@ -1,8 +1,9 @@
 
-
-name: pubmedDownload
 class: lathe
+name: pubmedDownload
 
-prep:
-  - commandLine: ./download.py -o ../../source/pubmed/baseline
-
+scripts:
```
Comment (kellrott) [inline transform/pubmed/download.yaml]: Leave this one as a prep command

**Review by kellrott — CHANGES_REQUESTED** [review]

**Comment by evergreen700** [comment]
Made the requested changes

**Comment by evergreen700** [comment]
PharmacoDB parse is on now!

**Pull request merged**